docs/using.rst

```diff
@@ -298,7 +298,7 @@ Bundle-Based Customization: Hooks
 
 Unlike handlers, hooks are part of the update bundle and must be
 specified in the bundle's :ref:`sec_ref_manifest` file and handled by a common
-executable.
+executable that should be part of the bundle.
```

Comment:
`part of the (update) bundle` is now at the beginning and end of the sentence, which sounds strange.